integrated_channels/blackboard/client.py

```diff
@@ -248,20 +265,6 @@ def _validate_full_course_metadata(channel_metadata_items):
                 HTTPStatus.NOT_FOUND.value
             )
 
-    def _resolve_blackboard_course_id(self, external_id):
```

Comment:
This seemed more like a helper method rather than a validation method so I moved it closer to the other helper methods.